CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ This file is used to list changes made in each version of the AWS ParallelCluste
 
 **CHANGES**
 - Ubuntu 20.04 is no longer supported.
+- Add SingleAvailabilityZone parameter to fleet_config.json
```

Comment:
Changelog entries must be more focus on the user experience than on the implementation details. A user does not care about where we inject the new parameter, but on the consequences of enabling it.